package.json

```
@@ -6,7 +6,9 @@
     "install:frontend": "cd frontend && npm install",
     "install:backend": "cd backend && npm install",
     "start:frontend": "cd frontend && npm start",
-    "start:server": "cd backend && npm start"
+    "start:server": "cd backend && npm start",
+    "test:run": "npx cypress run --headed",
```

Que tal?

```suggestion
    "cy:open": "cypress open",
```

Dessa forma, a Cypress App continua aberta após a execução dos testes.